### PR TITLE
DEPLOY-426: Declare the infrastructure chart as a requirement of the dbp charts

### PR DESCRIPTION
**Pull request by svidrascu**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,17 +78,78 @@ kubectl create -f secrets.yaml --namespace $DESIREDNAMESPACE
...
 --namespace=$DESIREDNAMESPACE
 
 #On AWS
 helm install alfresco-incubator/alfresco-dbp \
+--set alfresco-infrastructure.alfresco-api-gateway.keycloakURL="http://$ELBADDRESS/auth/" \
+--set alfresco-infrastructure.persistence.efs.enabled=true \
+--set alfresco-infrastructure.persistence.efs.dns="$NFSSERVER" \
 --set alfresco-content-services.repository.environment.IDENTITY_SERVICE_URI="http://$ELBADDRESS/auth" \
 --namespace=$DESIREDNAMESPACE
 ```
```
Comment (subaprasanna) [inline README.md]: @svidrascu  - we are missing instruction about persistence subpath i think .. not sure

Comment (svidrascu) [reply README.md]: We are not setting those in the default readme, just in the deploy script. The subpaths are already set.

**Comment by melahn** [comment]
I'm assuming the merge conflict with +version: 0.3.0-DEPLOY-426 will be resolved in favor of 0.4.0

**Comment by svidrascu** [comment]
Yes.

**Pull request merged**